vulpes/command/debug.cpp

```diff
@@ -56,6 +58,18 @@ bool get_object_id_from_network_id(std::vector<VulpesArg> input) {
     return true;
 }
 
+bool toggle_cprint_budget(std::vector<VulpesArg> input) {
```

Comment:
What happens if I turn on `get_network_id_from_obj_id` or `get_object_id_from_network_id` at the same time as this? Do they print on top of each other?

Reply:
Those are single use commands. So they'd print once and then get removed from screen.